integration-tests/scripts/local-testnet.sh

```diff
@@ -18,12 +18,19 @@ cloneDependencies(){
   mkdir "$TESTNET_DIR"
 
   git clone https://github.com/multiversx/mx-chain-go "$TESTNET_DIR/mx-chain-go"
-  cd $TESTNET_DIR/mx-chain-go
-  git checkout 32d0f017fa68add935a4f3a226c674c45d440835
-  cd ../..
+  checkoutStableVersion mx-chain-go rc/v1.6.0
```

Comment:
add a variable for the mx-chain-go branch/ stable

Reply:
Added variable as suggested